src/app/content/utils.ts

```diff
@@ -30,6 +30,24 @@ export const getContentPageReferences = (content: string) =>
       };
     });
 
+  // rap content
+  const rapMatches = (content.match(/.\/([a-z0-9-]+(@[\d.]+)):([a-z0-9-]+.xhtml)/g) || [])
+    .map((match) => {
+      const [bookMatch, pageMatch] = match.split(':');
+      const pageId = pageMatch.substr(0, 36);
+      const [bookId, bookVersion] = bookMatch.split('@');
+
+      return {
+        bookId: bookId.substr(2),
+        bookVersion,
+        match,
```

Comment:
The other one takes a substring here, are you sure this is ok?

Reply:
yeah the legacy ones, for some reason, the match comes like this: `"/contents/d0a86917-6447-4b49-bed1-efb5c352cc77` so the `substr` is to remove the `"` but for the rap ones that come like: `./13ac107a-f15f-49d2-97e8-60ab2e3b519c@29.7:99d38770-49c7-49d3-b567-88f393ffb4fe.xhtml` I don't thinks it's needed